docs/resources/css_cluster.md

```diff
@@ -187,6 +187,33 @@ The `volume` block supports:
 
 * `size` - (Required, Int) Specifies the volume size in GB, which must be a multiple of 10.
 
+* `volume_type` - (Required, String, ForceNew) Specifies the volume type. COMMON: Common I/O. The SATA disk is used.
+  HIGH: High I/O. The SAS disk is used. ULTRAHIGH: Ultra-high I/O. The solid-state drive (SSD) is used. Changing this
+  parameter will create a new resource.
+
+<a name="Css_ess_node_config_forceNew"></a>
+The `master_node_config` and `client_node_config` block supports:
+
+* `flavor` - (Required, String, ForceNew) Specifies the flavor name. For example: value range of flavor ess.spec-2u8g:
```

Comment:
the example should be only for `master_node_config` and `client_node_config` block.

Reply:
done